templates/assets/service-ec2.yml

```diff
@@ -401,27 +478,9 @@ Resources:
                 log_group_name = ${AWS::StackName}
           commands:
             codedeploy-install:
-              command: "./codedeploy-install auto"
+              command:
+                Fn::Sub: ./codedeploy-install auto --proxy http://${HttpProxy}
```

Comment:
Might need a condition called HasProxy to check if the HttpProxy value is blank, then do an If statement here to include or exclude the --proxy flag.

Reply:
Nice catch...I added the conditional for it.